core/src/main/java/com/novoda/noplayer/exoplayer/mediasource/ExoPlayerMappedTrackInfo.java

```diff
@@ -0,0 +1,25 @@
+package com.novoda.noplayer.exoplayer.mediasource;
+
+import com.google.android.exoplayer2.source.TrackGroupArray;
+import com.google.android.exoplayer2.trackselection.MappingTrackSelector;
+
+public class ExoPlayerMappedTrackInfo {
```

Comment:
There are lots of classes in the project that are public because are inside packages and they should not be. We will have to flatten all this classes and only expose what's needed. Not int his PR though.